rdflib/tools/csv2rdf.py

```diff
@@ -315,7 +314,7 @@ def __init__(self):
         self.COLUMNS = {}
         self.PROPS = {}
 
-        self.OUT = codecs.getwriter("utf-8")(sys.stdout, errors="replace")
+        self.OUT = sys.stdout
```

Comment:
Does this redirect properly to output files if an `-o` flag is used?

Reply:
It does. Added a test to ensure.